### DISCUSSION TITLE
Raise "started" event analogous to the "stopped" event

### DISCUSSION THREAD

**Discussion by tobilg** [bbc/sqs-consumer#387] [Feature Request]
### Background

We have some issues when creating integration tests, mostly during setup and cleanup. The docs at https://bbc.github.io/sqs-consumer/interfaces/Events.html state that there's an `stopped` event, but the equivalent `started` event is missing unfortunately. Would it be possible to add this?

### Objectives

1. Add a new `started` event

**Comment by nicholasgriffintn** [comment]
Thanks for the suggestion, I've created a ticket here: https://github.com/bbc/sqs-consumer/issues/388
